### ISSUE TITLE
Error with actually using Vysion

### ISSUE THREAD

**Issue by TedKaczynskisCousin** [epsilonfive/VYSION#3]
Hi im using a 5.4.0.0034 (whatever those last 4 digits mean) TI-84+ CE calculator, and whenever I run vysion from the Prgm menu I get a screen saying "hello welcome to vysion blah blah filesystem has been initialized" and then the ram resets.
However when I try to relaunch the program, the same thing happens. Nothing changes! I've reinstalled all the clibs on my calculator, and I have 152384 RAM free and 2812k Archive free.
I've also put the wallpapers in, but nothing seems to work!

Maybe it works a similar way to Cesium, i thought, so i checked the Apps menu. Nothing there. 

Can you help?

**Comment by epsilonfive** [comment]
Hey LordPhyre-
This is an unsolved issue that's been around for some time, and unfortunately hasn't been solved (it shows up intermittently on others' calcs, but always has worked on my 5.3.1.0058 CE).

Additionally, VYSION 1 is no longer receiving updates, as I've been working on rewriting the program from the ground up for the last 8 months or so, in part to solve these issues. You can follow the development of VYSION 2 [here](https://www.cemetech.net/forum/viewtopic.php?t=17202&postdays=0&postorder=asc&start=0).

So basically, I can't fix this issue (my apologies). I'd use Cesium until VYSION 2 is released.
